pages/session-expired.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Session Expired
+heading: Session expired
+layout: page.html
+---
+
+<div class="main home" role="main">
+  <div class="section main-menu">
+    <div class="row vads-u-padding-y--5">
+      <div class="usa-content small-12 columns">
```

Comment:
Is it worth removing the Foundation classes in here while you're at it?

Reply:
Removed!